udp_driver/src/udp_sender_node.cpp

```diff
@@ -141,3 +141,6 @@ void UdpSenderNode::subscriber_callback(std_msgs::msg::Int32::SharedPtr msg)
 
 }  // namespace udp_driver
 }  // namespace drivers
+
+#include <rclcpp_components/register_node_macro.hpp>  // NOLINT
```

Comment:
did you mean to include this in this PR? good catch all the same!

Reply:
I pushed this to your `generic-udp-nodes` branch instead. Thanks.